Mathlib/Analysis/Convex/LinearIsometry.lean

```diff
@@ -0,0 +1,74 @@
+/-
+Copyright (c) 2022 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+-/
+import Mathlib.Analysis.Convex.StrictConvexSpace
+import Mathlib.Analysis.Normed.Operator.LinearIsometry
+
+/-!
+# (Strict) convexity and linear isometries
+
+In this file we prove some basic lemmas about (strict) convexity and linear isometries.
+
+-/
+
+open Function Set Metric
+open scoped Convex
+
+section SeminormedAddCommGroup
+
+variable {𝕜 E F : Type*}
+  [NormedField 𝕜] [PartialOrder 𝕜]
+  [SeminormedAddCommGroup E] [NormedSpace 𝕜 E]
+  [SeminormedAddCommGroup F] [NormedSpace 𝕜 F]
+
+@[simp]
+lemma LinearIsometryEquiv.strictConvex_preimage {s : Set F} (e : E ≃ₗᵢ[𝕜] F) :
```

Comment:
I don't see why you prove this for linear isometries and not for continuous linear equivs (which would be stronger, as a linear isometry is a continuous linear equiv). Same thing for the next lemma.